README.md

```diff
@@ -19,6 +19,21 @@ With [vim-plug](https://github.com/junegunn/vim-plug) and many similar ones, you
 Plug 'ledger/vim-ledger'
 ```
 
+### Install using VIM packages functionality (vim 8+)
+
+To install as a single plugin, using VIM packages functionality first create a target folder:
+``` plugin
+mkdir -p ~/.vim/pack/ledger/start
+cd ~/.vim/pack/ledger/start
```

Comment:
Does vanilla vim have an `opt` or `filetype` instead of `start` that can be used to only load filetype detection automatically rather than always starting the plugin?

Reply:
No worries!  I perfectly understand if you do not want to include it.  I just started recently using vim and ledger and did try to follow the generic vim plugin documentation, rather than using a package manager.